solr/solrj/src/java/org/apache/solr/common/cloud/ZkStateReader.java

```diff
@@ -549,7 +636,7 @@ private void constructState(Set<String> changedCollections) {
       log.trace(
           "clusterStateSet: interesting [{}] watched [{}] lazy [{}] total [{}]",
           collectionWatches.keySet(),
-          watchedCollectionStates.keySet(),
+          collectionWatches.activeCollections().size(),
```

Comment:
This shouldn't print the size, it should print the collection names to match the rest of the intent of this trace statement.

Reply:
Thanks it was corrected